fixture-monkey/src/main/java/com/navercorp/fixturemonkey/arbitrary/ArbitraryTree.java

```diff
@@ -22,20 +22,23 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
-import java.util.Queue;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.function.Supplier;
 
 import javax.annotation.Nullable;
 
 import net.jqwik.api.Arbitrary;
 
+import com.navercorp.fixturemonkey.api.random.Randoms;
 import com.navercorp.fixturemonkey.generator.ArbitraryGenerator;
 import com.navercorp.fixturemonkey.validator.ArbitraryValidator;
 
+
+
```

Comment:
Blank lines should be removed if it is not necessary

Reply:
I could address this in a separate PR.

Reply:
@sandrawangyx 
Are there any reasons why you remove blank lines in separate PR?
Since we use `Squash and merge`, you do not need to care about commit logs.

Reply:
cool ty, fix now